starboard/extension/accessibility.h

```diff
@@ -0,0 +1,241 @@
+  // fields of SbAccessibilityCaptionSettings will ever contain this value.
+  kSbAccessibilityCaptionStateOverride,
+} SbAccessibilityCaptionState;
+
+// A group of settings related to system-level closed captioning settings, for
+// platforms that expose closed captioning settings.
+typedef struct SbAccessibilityCaptionSettings {
+  SbAccessibilityCaptionColor background_color;
+  SbAccessibilityCaptionState background_color_state;
+
+  SbAccessibilityCaptionOpacityPercentage background_opacity;
+  SbAccessibilityCaptionState background_opacity_state;
+
+  SbAccessibilityCaptionCharacterEdgeStyle character_edge_style;
+  SbAccessibilityCaptionState character_edge_style_state;
+
+  SbAccessibilityCaptionColor font_color;
+  SbAccessibilityCaptionState font_color_state;
+
+  SbAccessibilityCaptionFontFamily font_family;
+  SbAccessibilityCaptionState font_family_state;
+
+  SbAccessibilityCaptionOpacityPercentage font_opacity;
+  SbAccessibilityCaptionState font_opacity_state;
+
+  SbAccessibilityCaptionFontSizePercentage font_size;
+  SbAccessibilityCaptionState font_size_state;
+
+  SbAccessibilityCaptionColor window_color;
+  SbAccessibilityCaptionState window_color_state;
+
+  SbAccessibilityCaptionOpacityPercentage window_opacity;
+  SbAccessibilityCaptionState window_opacity_state;
+
+  // The |is_enabled| attribute determines if the user has chosen to enable
+  // closed captions on their system.
+  bool is_enabled;
+
+  // Some platforms support enabling or disabling captions, some support reading
+  // whether they are enabled from the system settings, and others support
+  // neither. As a result, there are separate checks for getting and setting
+  // the value that is contained in the |is_enabled| attribute. Modifying the
+  // attribute via |SbAccessibilitySetCaptionsEnabled| will change the setting
+  // system-wide. Attempting to read |is_enabled| when the value of
+  // |supports_is_enabled| is false will always return false. Attempting to set
+  // |is_enabled| via |SbAccessibilitySetCaptionsEnabled| when the value of
+  // |supports_set_enabled| is false will fail silently.
+  bool supports_is_enabled;
+  bool supports_set_enabled;
+
+  // Some platforms may specify that when setting a property, it should override
+  // data from video streams and application settings (unless the application
+  // has its own overrides). Depending on whether this attribute contains true
+  // or false, the values of |SbAccessibilityCaptionState| should be interpreted
+  // differently.
+  bool supports_override;
+} SbAccessibilityCaptionSettings;
+
+typedef struct CobaltExtensionAccessibilityAPI {
+  // Name should be the string
+  // |kCobaltExtensionAccessibilityName|. This helps to validate that
+  // the extension API is correct.
+  const char* name;
+
+  // This specifies the version of the API that is implemented.
+  uint32_t version;
+
+  // Get the platform settings related to the text-to-speech accessibility
```

Comment:
nit: please use "Gets" instead of "Get" since there should be an implied subject of "This function".

This applies to other function comments in this file, too.

Please see https://engdoc.corp.google.com/eng/doc/devguide/cpp/styleguide.md?cl=head#Function_Comments for more context.

Reply:
Done

Reply:
I don't have access to the link

Reply:
Sorry, please use this one: https://google.github.io/styleguide/cppguide.html#Function_Comments.